config/data_scrubber.go

```diff
@@ -49,6 +49,11 @@ func CompileStringsToRegex(words []string) []*regexp.Regexp {
 			continue
 		}
 
+		if word == "*" {
+			log.Warnf("data scrubber: %s not compiled. The sensitive word '*' is not supported", word)
```

Comment:
let's say `ignoring wildcard-only ("*") sensitive word as it is not supported` instead. Upon further review I'd change the message for the other warning in a similar way as "skipped" is more meaningful than "not compiled" to a user, IMO.